### ISSUE TITLE
Nothing happens after linking

### ISSUE THREAD

**Issue by githubnavigator** [signal-csharp/Signal-Windows#222]
I have installed from the Windows Store and successfully linked to my iPhone, and I gave permission to my contacts. However, no synchronization appears to occur. 

I see that the latest version in Windows store is 0.2.26, whereas the current release here is 0.2.30. It may be that version 0.2.30 links up properly, but I haven't been able to figure out how to turn it into an appxbundle (the release here on GitHub is in appxupload format). 

Any advice?

**Comment by golf1052** [comment]
Ah I didn't realize .appxupload can't be used to sideload. I've uploaded the .appxbundle.

**Comment by githubnavigator** [comment]
That is installable, but the certificate is not valid. Will you be uploading this to the Store? If so, I can just update it from there and report back on sync status.

**Comment by golf1052** [comment]
Yeah, I'm working on getting the new version uploaded to the store.

**Comment by KhaaL** [comment]
Hi! My issue could be related to this, or not. Giving it a shot nontheless!

Im on version 0.2.26 and after linking with my phone, I am getting a blank app page (see screenshot). Clicking on the "add contact" icon, takes me to the page to add a new contact, but crashes before letting enter anything.


Let me know if you need more information to make debugging easier (along with instructions on how to get said information☺).

![image](https://user-images.githubusercontent.com/692524/108701381-2a9c3300-7508-11eb-8697-0bd37e84b2d0.png)

**Comment by samsoncheong93** [comment]
Yeah I got the same issue too. Is the new version already uploaded to the store?

**Comment by golf1052** [comment]
Yes, 0.3.0 was uploaded to the store last week.

**Comment by samsoncheong93** [comment]
But I still see the same issue where the old messages were not synced to this app after linking, but sending messages were works fine. Is it actually API issues or is a bug instead?

**Comment by golf1052** [comment]
>But I still see the same issue where the old messages were not synced to this app after linking

I believe message syncing after linking is either broken ATM or was never implemented so it will need to be added/fixed.
